soc/arm/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -118,3 +118,11 @@ config NRF52_ANOMALY_132_DELAY_US
 	  depending on application, workaround may also need to be applied.
 	  Additional drivers initialization increases initialization time and delay
 	  may be shortened. Workaround is disabled by setting delay to 0.
+
+config NRF52_ANOMALY_198_WORKAROUND
+	bool "Anomaly 198 workaround"
+	default y
+	depends on SOC_NRF52840
```

Comment:
what about having `depends on NRFX_SPIM3` as well?

Reply:
Sounds sensible. Just added it.